docs/css/style.css

```diff
@@ -0,0 +1,894 @@
+@font-face {
```

Comment:
No, what I'm saying is the script that pushes docs to gh pages can also just compile the CSS on its own so that it doesn't need to pollute the master branch.